src/test/java/carracing/CarRacingWinnersTest.java

```diff
@@ -0,0 +1,27 @@
+package carracing;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CarRacingWinnersTest {
+
+    @DisplayName("자동차 경주 게임을 완료한 후 우승자를 찾는다.")
+    @Test
+    void find_winners_after_carRacing_is_done() {
+        Car car1 = new Car("moon");
+        car1.move(4);
+        car1.move(4);
+        Car car2 = new Car("zi");
```

Comment:
테스트코드에서도 의미 있는 변수명을 사용하면 어떨까요?
자동차의 이름대로 변수명을 지어줘도 괜찮을 것 같아요.
```suggestion
        Car moon = new Car("moon");
        moon.move(4);
        moon.move(4);
        Car zi = new Car("zi");
```